solr/test-framework/src/java/org/apache/solr/cloud/MiniSolrCloudCluster.java

```diff
@@ -89,6 +90,10 @@
 public class MiniSolrCloudCluster {
 
   private static final Logger log = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
+  private static final URL PRE_GENERATED_PRIVATE_K_URL =
```

Comment:
nit-pick, but can we use `KEY` instead of `K` i.e. `PRE_GENERATED_PRIVATE_KEY_URL`